src/Options.php

```diff
@@ -143,6 +149,27 @@ public function getTracesSampleRate(): ?float
         return $this->options['traces_sample_rate'];
     }
 
+    /**
+     * Sets if tracing should be enabled or not. If null tracesSampleRate takes
+     * precedence.
+     *
+     * @param ?bool $enableTracing Boolean if tracing should be enabled or not
+     */
+    public function setEnableTracing(?bool $enableTracing): void
+    {
+        $options = array_merge($this->options, ['enable_tracing' => $enableTracing]);
+
+        $this->options = $this->resolver->resolve($options);
+    }
+
+    /**
+     * Gets if tracing is enabled or not.
+     */
+    public function getEnableTracing(): ?bool
+    {
+        return $this->options['enable_tracing'];
```

Comment:
You can run `vendor/bin/phpstan -b` to update the phpstan baseline file.

Reply:
Yeah, maybe true - it is a bit confusing.
The thing is the logic behind how `isTracingEnabled` behaves works a bit differently than just a simple boolean.
This getter however really returns the plain content.